src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -45,16 +45,22 @@
     private static final ObjectMapper mapper = new ObjectMapper();
     private static final Logger LOG = LoggerFactory.getLogger(Auth0Users.class);
 
+    private static URI getUrl(String searchQuery, int page, int perPage, boolean includeTotals) {
+        return getUrl(searchQuery, page, perPage, includeTotals, true);
+    }
+
     /**
      * Constructs a user search query URL.
      * @param searchQuery   search query to perform (null value implies default query)
      * @param page          which page of users to return
      * @param perPage       number of users to return per page
      * @param includeTotals whether to include the total number of users in search results
+     * @param includeDefaultQuery whether to include the default query in the search criteria
```

Comment:
This param name might make more sense if it were something like `limitToCurrentUsers`. Maybe there's a better name, but "default query" does not give a great sense of what's actually happening under the hood.

Reply:
Updated.